wordpress-mobile/WordPress-iOS
# Wires the blogging reminders scheduler to the UI.

## What to test:

Test that your day selection is persisted for each blog you  configure.
Test that changing the calendar locale between executions of the reminders flow doesn't affect the day selection.

## What to ignore:

Even though the notifications scheduling is wired, let's ignore testing notifications for now.  Since this feature is behind a flag, we can make this call safely, and focus on testing the notifications at a future time.

## To test:

Test the blogging reminders flow a few times on the same blog, then switch to another and come back to the original one.  The day selection should always be persisted across executions of the flow.

Also, try:

1. Set the device region to "United States"
2. Start the reminders flow.
3. Make sure "Sunday" is the first day shown in the day picker.
4. Select the days for your reminders
5. Complete the flow, so that the reminders are stored.
6. Change the region of your device to "Argentina"
7. Re-run the blogging reminders flow
8. Make sure the first day shown in the day picker is "Monday"
9. Make sure the day selection is correct and matches what it was when the region was "United States".

## Regression Notes

1. Potential unintended areas of impact

None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
